### DISCUSSION TITLE
Feedback request for GSoC 2024( In case anybody has spare time, I realize I am late)

### DISCUSSION THREAD

**Discussion by SampurnaM** [MDAnalysis/mdanalysis#4556] [GSoC Discussions]
Hi,
I have only just managed to upload the PDF with Google Doc link in the GSoC site.
I understand this is a very short turnaround time , and grateful if I can get feedback, also just happy for the opportunity to do this, if not.                                                                                  Thanks, 
Sampurna

**Comment by RMeli** [comment]
Thank you for participating in GSOC 2024 with us. Whether you were selected as a GSoC contributor or not, you are a valued member of the MDAnalysis community and we are excited to work with you outside of the GSoC program as well!
